### PR TITLE
Add support for AMIgen7's -g and -m Flagged-options

### LINKED ISSUE #380: Enable usage of AMIgen7's `-g` and `-m` Flagged-options
**Current Behavior:**

The `-g` and `-m` Flagged-options are new. Need to update spel to allow for their use.

**Desired behavior:**

Add new (or update existing) builders to take advantage of AMIgen7's (pending merge of plus3it/AMIgen7#86) `-g` and `-m` Flagged-options

### PR DESCRIPTION
**Pull request by ferricoxide**
Closes #380 

Validated using activators for `-g core` and `-m /path/to/GnomeDesktop.pkgs`. 

**Note:** Experienced failures with `-g <NOT_CORE>`. However, the failure is not due to issues within spel. The issues will need to addressed in AMIgen7 (see plus3it/AMIgen7#87), directly.

### CODE REVIEW

**Comment by ferricoxide** [comment]
Also...

go codebuild go

**Comment by ferricoxide** [comment]
go codebuild go

File: spel/scripts/amigen7-build.sh [hunk 1]
```diff
@@ -205,6 +207,18 @@ bash -eux -o pipefail "${ELBUILD}"/MkChrootTree.sh "${DEVNODE}" "" "${AMIGENSTOR
 echo "Executing MkTabs.sh"
 bash -eux -o pipefail "${ELBUILD}"/MkTabs.sh "${DEVNODE}"
 
+# Construct the cli option string for alternate-manifest
+CLIOPT_ALTMANIFEST=""
+if [[ -n ${AMIGENMANFST} ]]
```
Comment (lorengordon) [inline spel/scripts/amigen7-build.sh]: I'm surprised shellcheck didn't flag this...
```suggestion
if [[ -n "${AMIGENMANFST}" ]]
```

Comment (ferricoxide) [reply spel/scripts/amigen7-build.sh]: Doublequotes is mostly for `[ ]` test blocks. Doublequotes inside `[[ ]]` testblocks is only when you specifically need them for the RHS of the test (e.g. most frequently where the RHS is `""` or something with spaces in it).

**Pull request merged**